src/groups/Sentence.jsx

```diff
@@ -29,147 +30,73 @@ class Sentence extends React.Component {
-        accept ratio for comments they submitted,
-      `);
+
+  isDefault(filterName) {
+    let isDefault = true;
+    const f = this.props[filterName];
+    const maxDifferent = f.userMax !== f.max && f.userMax < f.max;
+    const minDifferent = f.userMin !== f.min;
+    if (maxDifferent || minDifferent) {
+      isDefault = false;
     }
+    return isDefault;
   }
-  makeRepliesPerCommentPhrase() {
-    const repliesPerCommentMaxDefault = 0;
-    const repliesPerCommentMinDefault = 1;
-    /* if the min and max for comments ARE NOT the defaults, render. */
-    if (
-      this.props['stats.replies_per_comment'].userMin !== repliesPerCommentMaxDefault ||
-      this.props['stats.replies_per_comment'].userMax !== repliesPerCommentMinDefault
-    ) {
-      return (`
-        between
-        ${this.props['stats.replies_per_comment'].userMin}
-        and
-        ${this.props['stats.replies_per_comment'].userMax}
-        replies per comment on average,
-      `);
-    }
+
+  formatName(name) {
+    name = name.replace("-", " ")
+    return name.replace(/\w\S*/g, function(txt){return txt.charAt(0).toUpperCase() + txt.substr(1).toLowerCase();});
   }
-  makeRepliesPhrase() {
-    const repliesMaxDefault = 0;
-    const repliesMinDefault = 1000;
-    /* if the min and max for comments ARE NOT the defaults, render. */
-    if (
-      this.props['stats.replies'].userMin !== repliesMaxDefault ||
-      this.props['stats.replies'].userMax !== repliesMinDefault
-    ) {
-      return (`
-        between
-        ${this.props['stats.replies'].userMin}
-        and
-        ${this.props['stats.replies'].userMax}
-        replies to their comments in total,
-      `);
+
+  getSpecific() {
+    let breakdown = " in all authors and sections";
+    if (this.props.specificBreakdown) {
+      breakdown = (
+        <span>
+        {` users who submitted comments to ${this.props.breakdown} ${this.formatName(this.props.specificBreakdown)} `}
+        </span>
+      );
     }
+    return breakdown;
   }
 
-  getClauses() {
-    const matchCommands = _.filter(this.props.queries[0].commands, command => {
-      return _.has(command, '$match');
-    });
-
-    // this code is brittle and will break in the future.
-    // sorry whoever is fixing this.
-    // you probably changed the filter.field name, or there are multiple
-    // filters with the same trailing segment
-    return _.map(matchCommands, command => {
-      const name = _.first(_.keys(command.$match));
-      const baseFilter = _.find(window.filters, filter => {
-        return filter.field === _.last(name.split('.'));
-      });
-      let clause;
-
-      if (_.has(command.$match[name], '$gte')) {
-        clause = `between ${command.$match[name].$gte}`;
-      } else if (_.has(command.$match[name], '$lte')) {
-        clause = `and ${command.$match[name].$lte} ${baseFilter.description},`;
+  getFilters() {
+    let nonDefaultFilters = [];
+    this.props.filterList.map((filterName) => {
```

Comment:
maybe we should have a chain here or something so it's more clear what's going on, as opposed to having an extra array just to hold stuff.

The first step would just be a filter like `this.props.filterList.filter(name => !this.isDefault(filterName))`